internal/cluster/validations/validations.go

```diff
@@ -398,6 +399,12 @@ func ValidateClusterCreateIPAddresses(ipV6Supported bool, clusterId strfmt.UUID,
 		return common.NewApiError(http.StatusBadRequest, err)
 	}
 
+	if (len(params.APIVips) > 1 || len(params.IngressVips) > 1) && !featuresupport.IsFeatureSupported(*params.OpenshiftVersion,
+		models.FeatureSupportLevelFeaturesItems0FeatureIDDUALSTACKVIPS) {
+
+		return common.NewApiError(http.StatusBadRequest, errors.Errorf("%s %s", "dual-stack VIPs are not supported in OpenShift", *params.OpenshiftVersion))
```

Comment:
A suggestion, should this response also include the OpenShift version from which dual stack is supported?

Reply:
It's not trivial because I want to leverage `featuresupport.IsFeatureSupported()` function without hardcoding anything. I could write manually "use 4.12 or later" but then I defeat the purpose of using the `featuresupport` package.

I'm happy to do what you suggest, but I'd need some direction how to do it without any hardcoding here